content/docs/getting-started/installation/helm/modules/authorizationv2-0.md

```diff
@@ -286,20 +390,25 @@ spec:
+| conjur.secretPath                   | The secret path of the Conjur secretProviderClass config secret object.                                                             | No       |                                             |-                                                                                                                            |
 | **storageSystemCredentials**        | This section configures the storageSystemCredentials.                                                                  | -        | -                                                                                                                            |
 | **secretProviderClasses**           | This section configures secretProviderClass objects.                                                                   | Yes      | -                                                                                                                            |
 | vault                               | A list of secretProviderClass objects when using Vault.                                                                | Yes      | -                                                                                                                            |
 | conjur                              | A list of secretProviderClass objects when using Conjur.                                                               | No       | -                                                                                                                            |
 | conjur.name                         | The name of a Conjur secretProviderClass object.                                                                       | No       | -                                                                                                                            |
 | conjur.paths                        | The secret paths of a Conjur secretProviderClass object.                                                               | No       | -                                                                                                                            |
-| **secrets**                         | This section configures Kubernetest secrets with their names.                                                          | No      | -                                                                                                                            |
+| **secrets**                         | This section configures Kubernetes secrets with their names.                                                          | No      | -                                                                                                                            |
 {{< /collapse >}}
 </ul>
 
```

Comment:
`7. Install the driver using `helm`:`

Could you update 'driver' to 'module' as this is for Auth install?